osbs/utils.py

```diff
@@ -259,7 +260,7 @@ def git_repo_humanish_part_from_uri(git_uri):
     elif git_uri.endswith(".git"):
         git_uri = git_uri[:-4]
 
-    return os.path.basename(git_uri)
+    return urlparse(git_uri)[2].lstrip('/')
```

Comment:
`urlparse` returns a named tuple, maybe use `.path` instead of `[2]`?

But! Should we actually do this? The length of build config names are quite short, this will cause a bunch of BuildConfigs to start with either rpms/... or containers/... which seems wasteful and not very intuitive.

I'm thinking that this shouldn't change at all. The `git_repo_humanish_part_from_uri` is used to determine the value for label `git-repo-name`. If we change processing of this value, it means that on the next code release, osbs-client won't be able to find **any** existing BuildConfigs.

Instead, we should change `make_name_from_git` to take the full repo name into account. Maybe just for generating the hash?

Reply:
You were the one who said to change the names. And didn't we just have a discussion about the fact that we get existing configs by labels, not names?
It's easy enough to change the hash code if that's what we want to do.
```
-     full = repo + branch
+    full = urlparse(repo).path.lstrip('/') + branch
```

Reply:
I suggested changing the generated BuildConfig name, not the `git-repo-name` label.

Reply:
Fixed.